modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-order-impl/rest-openapi.yaml

```diff
@@ -633,10 +633,17 @@ components:
                     minimum: 0
                     readOnly: true
                     type: integer
+                itemsCount:
+                    format: int32
+                    readOnly: true
+                    type: integer
                 modifiedDate:
                     format: date
                     readOnly: true
                     type: string
+                oneLineAddress:
```

Comment:
Just an observation: this could be useful elsewhere too, good we now have it.

Reply:
Yes, we already have it indexed using the `CommerceAddressFormatter`